pwa/src/templates/templateParts/header/HeaderTemplate.tsx

```diff
@@ -122,13 +132,22 @@ export const HeaderTemplate: React.FC<HeaderTemplateProps> = ({ layoutClassName
       },
       icon: <GitHubLogo />,
     },
-    {
-      label: "Login",
-      handleClick: () => {
-        navigate("/login");
-      },
-      icon: <FontAwesomeIcon icon={faCircleUser} />,
-    },
+
+    isLoggedIn()
+      ? {
+          label: "Logout",
+          handleClick: () => {
+            navigate("/logout");
+          },
+          icon: <FontAwesomeIcon icon={faCircleUser} />,
+        }
+      : {
+          label: "Login",
+          handleClick: () => {
+            navigate("/login");
+          },
+          icon: <FontAwesomeIcon icon={faCircleUser} />,
+        },
```

Comment:
I'd suggest to remove this ternary expression, there are a couple of ways to do this, but maybe the following would be very usable:

```
const logoutMenuItem = { ... item comes here };
const loginMenuItem = { ... item comes here };

const secondaryTopNavItems = [ ... always available items here ];

if (isLoggedIn()) secondaryTopNavItems.push(logoutMenuItem);
if (!isLoggedIn()) secondaryTopNavItems.push(loginMenuItem);
```